tests/app/main/views/test_admin_manager.py

```diff
@@ -204,3 +230,143 @@ def test_successful_post_flashes(self, data_api_client, send_user_account_email)
         res = self.client.post('/admin/admin-users/invite', data={'role': 'admin', 'email_address': 'test@test.com'})
         assert res.status_code == 302
         assert_flashes(self, 'An invitation has been sent to test@test.com.', 'success')
+
+
+@mock.patch("app.main.views.admin_manager.data_api_client")
+class TestAdminManagerEditsAdminUsers(LoggedInApplicationTest):
+
+    admin_user_to_edit = {
+        "users": {
+            "active": True,
+            "emailAddress": "reality.auditor@digital.cabinet-office.gov.uk",
+            "id": 2345,
+            "locked": False,
+            "name": "Auditor of Reality",
+            "role": "admin-ccs-category",
+            "updatedAt": "2017-11-22T14:42:27.043468Z"
+        }
+    }
+
+    def test_should_load_edit_admin_user_page_heading_and_submit_button_correctly(self, data_api_client):
+        self.user_role = "admin-manager"
+        data_api_client.get_user.return_value = self.admin_user_to_edit
+
+        response = self.client.get("/admin/admin-users/2345/edit")
+        assert response.status_code == 200
+
+        document = html.fromstring(response.get_data(as_text=True))
+
+        assert document.xpath('//h1')[0].text.strip() == "reality.auditor@digital.cabinet-office.gov.uk"
+        assert document.xpath('//input[@class="button-save"]')[0].value.strip() == "Update user"
+
+    def test_should_correctly_load_and_prefill_edit_name_question(self, data_api_client):
```

Comment:
In general I'd avoid test names that say `should_correctly_do_something` unless it's 100% obvious what the correct behaviour is. 

I like the format `test_thing_x_does_action_y_under_condition_z` with a bit of common sense and flexibility in ordering/grammar (usually so it fits on one line!). If condition z is the default, that probably doesn't need to go in the test name, for example. 

So here it would be something like `test_edit_admin_user_form_prefills_edit_name_with_user_name` (and for the test below, `test_edit_admin_user_form_prefils_permission_for_category_user` would do).